202404-big-data/2024-04-big-data.qmd

```diff
@@ -0,0 +1,856 @@
+---
+title: "Wrangling larger-than-memory data in R"
+author: "Renata Diaz"
+format: 
+  uaz-revealjs:
+      logo: "_extensions/cct-datascience/uaz/logo.png"
+editor: source
+execute:
+  freeze: true
+  echo: true
+---
+
+## ⏰ Schedule ⏰ {.smaller}
+
+::: notes
+remember to record
+:::
+
+| Topic                                    | Time          |
+|:-----------------------------------------|:--------------|
+| Welcome & intros                         | 11:00 - 11:05 |
+| (Really) big data in R: the landscape    | 11:05 - 11:10 |
+| Data storage and access with `arrow`     | 11:15 - 11:30 |
+| `dplyr` wrangling with `arrow`           | 11:35 - 11:50 |
+| BREAK                                    | 11:50 - 12:00 |
+| `arrow` with `duckdb | 12:00 - 12:20 |
+| `duckdb` and raw SQL                   | 12:20-12:35   |
+| When to use what                         | 12:35-12:45   |
+| Wrap up                  | 12:45-1      |
+
+# Welcome!
+
+## CCT Data Science Team
+
+#### Supporting reproducible research, data pipelines, and software development in ALVSCE
+
+::: incremental
+-   Workshops
+-   Drop-in hours
+-   "Incubator" program
+-   Longer-term collaborations
+-   Learn more: <https://datascience.cct.arizona.edu>
+:::
+
+## Before we get started
+
+::: incremental
+-   These slides (and links!) are available at: NEW LINK HERE
+-   Please make sure you have R and RStudio installed
+-   Please install the following packages: `duckdb, arrow, dplyr, tidyr, dbplyr, DBI, here`
```

Comment:
If I recall correctly, arrow installation can take quite some time, especially if you don't already have arrow system library already.  Might want to get this going *before* talking about what we do?

Reply:
Good note! I think, because there's still quite a bit of talking before we actually *use* arrow, I might keep it here just to keep an organized flow. (I also suggested people install packages ahead of time in the reminder email.)